src/components/ids-popup-menu/ids-popup-menu.ts

```diff
@@ -155,12 +161,35 @@ export default class IdsPopupMenu extends Base {
       if (!this.parentMenuItem) {
         this.triggerEvent('hide', this, e);
       }
+      this.#mo?.disconnect();
     });
 
     // Set up all the events specifically-related to the "trigger" type
     this.refreshTriggerEvents();
   }
 
+  /** Hide any older (enterprise) menus */
+  hideOtherMenus() {
+    const openMenu = (document.querySelector('.popupmenu.is-open') as HTMLElement);
+    openMenu?.classList.remove('is-open');
+    const openMenuElem = (document.querySelector('[data-popupmenu].is-open') as HTMLElement);
+    openMenuElem?.classList.remove('is-open');
```

Comment:
I think it's better to fire the [popupmenu's `close()` API](https://github.com/infor-design/enterprise/blob/main/src/components/popupmenu/popupmenu.js#L2384-L2390):
```suggestion
    openMenuElem?.data('popupmenu')?.close(true, true);
```

I think this is necessary because there are events bound during `open()/close()`, among other component interactions that wouldn't be accounted for simply by using the class to hide.

Reply:
@EdwardCoyle while this would be better it would require using the jQuery API and introduce a dependency. Since this is about cancelling it only I think it would be fine so i did it this way.